pkg/common/helper.go

```diff
@@ -475,6 +477,29 @@ func CatchAndLogPanicWithOptions(ctx context.Context,
 	return nil
 }
 
+// MapStringToStringMatchByLabelSelectorStr return whether labelsMap matched by encoded label selector
+// corresponding to https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
+//
+// Example:
+//   labelsMap:
+//     a: b
+//     c: d
+//   encodedLabelSelector: a=b
+// returns true
+func MapStringToStringMatchByLabelSelectorStr(labelSelector string,
```

Comment:
```suggestion
func LabelsMapMatchByLabelSelector(labelSelector string,
```
It's a mouthful 🤷 
I know `MapStringToString` is more generic, but then you say `LabelSelector` so... we're talking labels here anyways